src/pages/recentDeliveries/RecentDeliveriesPage.tsx

```diff
@@ -0,0 +1,71 @@
+import "./RecentDeliveries.css";
+import Container from "react-bootstrap/Container";
+import RecentDeliveriesTable from "./RecentDeliveriesTable";
+import { Request, ExampleQueryDocument } from "../../generated/graphql";
+import { useQuery } from "@apollo/client";
+
+export class RequestWithDate {
```

Comment:
Using classes in js is frowned upon. You should try to use functional approach.  We can discuss this.